src/auto-complete-element.ts

```diff
@@ -1,10 +1,14 @@
 import Autocomplete from './autocomplete.js'
 import AutocompleteEvent from './auto-complete-event.js'
+import {fragment} from "./send";
 
 const state = new WeakMap()
 
 // eslint-disable-next-line custom-elements/file-name-matches-element
 export default class AutocompleteElement extends HTMLElement {
+  _fetchResult?: (el: Element, url: string) => Promise<string>;
```

Comment:
is this meant to be a private field? [Can use the private class field syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields)
```suggestion
  #fetchResult?: (el: Element, url: string) => Promise<string>;
```

Reply:
done :heavy_check_mark: thanks !